features/unit/v2indexer_rekey_responses.feature

```diff
@@ -0,0 +1,20 @@
+@unit
+@unit.indexer.rekey
+Feature: Indexer Client v2 Responses
+  Scenario Outline: SearchForAccounts response, authorizing address
+    Given mock http responses in "<jsonfiles>" loaded from "<directory>"
+    When we make any SearchAccounts call
+    Then expect error string to contain "<err>"
+    And the parsed SearchAccounts response should be valid on round <roundNum> and the array should be of len <len> and the element at index <index> should have authorizing address "<authaddr>"
```

Comment:
Add an improved `Then the re-serialized object equals the response in "<jsonfiles>" loaded from "<directory>"`

We can take the SDK object, serialize it back to JSON, and compare it to the input file.

This way we exactly compare all fields and don't need to re-implement the check for every test.

Reply:
In the Java SDK I was able to implement all response verify functions with the same helper:
```
  @Then("the parsed SearchForAssets response should be valid on round {long} "
      + "and the array should be of len {long} "
      + "and the element at index {long} "
      + "should have asset index {long}")
  public void the_parsed_SearchForAssets_response_should_be_valid_on_round_and_the_array_should_be_of_len_and_the_element_at_index_should_have_asset_index(
      Long int1, Long int2, Long int3, Long int4) throws IOException {
    verifyResponse(assetsResponse, shared.bodyFile);
  }
```

Reply:
I'm not sure how easy it is to directly serialize/deserialize the jsons using the tools the SDKs expose to a user dev. I'll take a whack at it!